src/javascript/CKEditor/JahiaClassicEditor.js

```diff
@@ -7,27 +7,18 @@ import {isElement} from 'lodash-es';
 
 import 'ckeditor5/ckeditor5.css';
 
-const productivityPluginsNameAndKey = {
-    FormatPainter: true,
-    formatPainter: true,
-    ExportPdf: true,
-    exportPdf: true,
-    ExportWord: true,
-    exportWord: true
-};
-
 export class JahiaClassicEditor extends ClassicEditor {
     static create(sourceElementOrData, config = {}) {
         config = {...JahiaClassicEditor.defaultConfig, ...config};
 
-        // eslint-disable-next-line
+        // eslint-disable-next-line no-undef
         const isProductivityEnabled = window?.contextJsParameters?.valid && CKEDITOR_PRODUCTIVITY_LICENSE;
-        // eslint-disable-next-line
+        // eslint-disable-next-line no-undef
         config.licenseKey = isProductivityEnabled ? CKEDITOR_PRODUCTIVITY_LICENSE : 'GPL';
 
         if (!isProductivityEnabled) {
-            JahiaClassicEditor.builtinPlugins = JahiaClassicEditor.builtinPlugins.filter(p => !productivityPluginsNameAndKey[p.pluginName]);
-            config.toolbar.items = config.toolbar.items.filter(i => !productivityPluginsNameAndKey[i]);
```

Comment:
no need to filter toolbar items as they are already filtered when plugin doesn't exist